public/keymaps/k/kapl_rev1_default.json

```
@@ -0,0 +1,29 @@
+{
+  "keyboard": "kapl/rev1",
+  "keymap": "default",
+  "commit": "71a48a01023449f9559f41d751fbedbdadfb1c46",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+      "KC_ESC",  "KC_1",    "KC_2",    "KC_3",  "KC_4",  "KC_5", "KC_PGUP",             "KC_HOME", "KC_6", "KC_7",  "KC_8",    "KC_9",    "KC_0",    "KC_MINS",
+      "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",  "KC_R",  "KC_T", "KC_PGDN",             "KC_END",  "KC_Y", "KC_U",  "KC_I",    "KC_O",    "KC_P",    "KC_EQL",
+      "KC_GRV",  "KC_A",    "KC_S",    "KC_D",  "KC_F",  "KC_G", "KC_LBRC",             "KC_RBRC", "KC_H", "KC_J",  "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",
+      "KC_LSFT", "KC_Z",    "KC_X",    "KC_C",  "KC_V",  "KC_B", "KC_EXLM",             "KC_QUES", "KC_N", "KC_M",  "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT",
+      "KC_LCTL", "KC_LGUI", "KC_LALT", "MO(2)", "MO(1)",    "KC_BSPC", "KC_DEL",     "KC_ENT", "KC_SPC",   "MO(1)", "KC_LEFT", "KC_DOWN", "KC_UP",   "KC_RGHT"
```

It's preferred to match these as closely to the `qmk_firmware` versions as possible.

```suggestion
      "KC_LCTL", "KC_LGUI", "KC_LALT", "MO(2)", "LT(1,KC_SPC)",    "KC_BSPC", "KC_DEL",     "KC_ENT", "KC_SPC",   "MO(1)", "KC_LEFT", "KC_DOWN", "KC_UP",   "KC_RGHT"
```